src/lib.rs

```diff
@@ -247,3 +249,39 @@ pub fn from_string(p: String) -> std::result::Result<String, JsValue> {
         .pretty_print(&map)
         .map_err(|e| raw_to_parse_error(&map, *e).to_string())?)
 }
+
+#[cfg_attr(feature = "profiling", inline(never))]
+#[cfg_attr(not(feature = "profiling"), inline)]
+#[cfg(not(feature = "wasm"))]
+pub fn from_path_with_load_paths(p: &str, loadpaths: &[&Path]) -> Result<String> {
```

Comment:
I believe you could just extend `from_path`. Rename that to `from_paths` rather than creating a new function and not using the old one